vexim/siteaddsubmit.php

```diff
@@ -143,25 +139,10 @@
       die;
     }
   } else if ($_POST['type'] == "alias") {
-    $query = "SELECT domain_id FROM domains
```

Comment:
I changed the `INSERT`-query that it gets the `domain_id` as a result from the `domains`-table